src/lib/remote-place-cache.ts

```diff
@@ -31,8 +24,38 @@ export type RemotePlace = {
 };
 
 export default class RemotePlaceCache {
-  private static cache: RemotePlaceDatastore = {};
+  private static cache: NodeCache;
+  private static readonly CACHE_TTL = 3600;
+  private static readonly CACHE_CHECK_PERIOD = 120;
+  
+  public static async getPlacesWithType(chtApi: ChtApi, contactType: ContactType, hierarchyLevel: HierarchyConstraint)
```

Comment:
what the difference between `public getPlacesWithType` and `public getRemotePlaces` in this class? are two interfaces like this required?